app/assets/javascripts/school_administrator_dashboard/main.jsx

```diff
@@ -1,9 +1,9 @@
-import PageContainer from  './page_container.jsx';
+import SchoolwideAttendance from  './schoolwide_attendance.jsx';
```

Comment:
@edavidsonsawyer Love this naming change! Much more precise. This is great, and as they say, naming things is one of the hardest problems in programming! 🌈